src/angular/usermenu/usermenu.scss

```diff
@@ -149,12 +149,12 @@
     display: inline-block;
   }
 
-  color: var(--sbb-color-metal);
+  color: var(--sbb-usermenu-text-color);
   font-size: var(--sbb-font-size-small);
   line-height: calc(#{sbb.pxToRem(14)} * var(--sbb-scaling-factor));
 
   :where(html.sbb-lean) & {
```

Comment:
Try removing this `:where(html.sbb-lean)` block and modify the colors for the lean variant in the `_typography.scss` instead.